src/main/java/uk/gov/companieshouse/api/strikeoffobjections/controller/ObjectionController.java

```diff
@@ -145,8 +146,12 @@ public ObjectionController(PluggableResponseEntityFactory responseEntityFactory,
         );
 
         try {
-            String objectionId = objectionService.createObjection(requestId, companyNumber, ericUserId, ericUserDetails);
-            ObjectionResponseDTO response = new ObjectionResponseDTO(objectionId);
+            Objection objection = objectionService.createObjection(requestId, companyNumber, ericUserId, ericUserDetails);
+            ObjectionResponseDTO response = objectionMapper.objectionEntityToObjectionResponseDTO(objection);
```

Comment:
we don't need to call the mapper, this will break the api spec as it looks like it will now return a full objection object in json in the http response body. The objection mapper will map all the fields. For successful, we should just return the id (see previous version of this method) and if ineligible we want just the status. Can't we just add status into the ObjectionResponseDTO constructor and populate it with the status from the returned Objection?

Reply:
In all the excitment of making the other changes forgot all about the spec - reworked this.